src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -76,6 +76,12 @@
 
   <Target Name="PublishNetSdks"
           BeforeTargets="Build">
+    <ItemGroup>
```

Comment:
why websdk is singled out

Reply:
I don't know, @MichaelSimons do you?

Reply:
It seems like it would be preferable to have project references to these from redist.csproj (with ReferenceOutputAssembly set to false).